client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-ssh-share-access/hooks/use-rotating-loading-messages.ts

```diff
@@ -0,0 +1,52 @@
+import { useTranslate } from 'i18n-calypso';
+import { useState, useEffect, useMemo } from 'react';
+
+interface UseRotatingLoadingMessagesOptions {
+	isStartingMigration: boolean;
+	migrationStarted: boolean;
+	shouldStartMigration: boolean;
+	rotationInterval?: number;
+}
+
+export const useRotatingLoadingMessages = ( {
```

Comment:
It's not clear to me why we need to pass `isStartingMigration`, `migrationStarted`, and `shouldStartMigration` to this component since they're only used to set `isBusy`.

Since `isBusy` is returned to the parent component, would it make more sense to set `isBusy` in the parent and pass just `isBusy` to `useRotatingLoadingMessages`?